### PR TITLE
Postpone QueueMap initialization until activation of the counters

### PR DESCRIPTION
**Pull request by pavel-shirshov**
**What I did**
I postponed generation of the QueueMaps until the counters are activated. 

**Why I did it**
We don't need QueueMaps until the counters are active

**How I verified it**
Build an image and run on DUT

**Details if related**

### CODE REVIEW

File: orchagent/flexcounterorch.cpp [hunk 1]
```diff
@@ -70,6 +70,10 @@ void FlexCounterOrch::doTask(Consumer &consumer)
                 }
                 else if(field == FLEX_COUNTER_STATUS_FIELD)
                 {
+                    // Currently the counters are disabled by default
+                    // The queue maps will be generated as soon as counters are enabled
+                    gPortsOrch->generateQueueMap();
```
Comment (sihuihan88) [inline orchagent/flexcounterorch.cpp]: Is the map generated after queue counters are enabled or all the counters are enabled?

File: orchagent/portsorch.cpp [hunk 2]
```diff
@@ -1817,61 +1817,6 @@ void PortsOrch::initializeQueues(Port &port)
     }
 
     SWSS_LOG_INFO("Get queues for port %s", port.m_alias.c_str());
-
-    /* Create the Queue map in the Counter DB */
-    /* Add stat counters to flex_counter */
```
Comment (lguohan) [inline orchagent/portsorch.cpp]: do we need queue map for applying qos configuration?

File: orchagent/flexcounterorch.cpp (hunk 1 above)
Comment (pavel-shirshov) [reply orchagent/flexcounterorch.cpp]: The map is generated after activation of the flex counters. As I understand the maps are related to port queues. So they will be used with queue counters.

File: orchagent/portsorch.cpp (hunk 2 above)
Comment (pavel-shirshov) [reply orchagent/portsorch.cpp]: No. We use only Port::m_queue_ids vector. We use the maps only:
1. swss: pfc handler
2. swss: countercheckorch
3. queuestat
4. snmp

**Pull request merged**